build_rules.gradle

```diff
@@ -518,24 +518,33 @@ ext.applyAvroNature = {
 
 // A class defining the set of configurable properties for createJavaQuickstartValidationTask
 class JavaQuickstartConfiguration {
```

Comment:
rename to  `JavaReleaseValidationConfiguration`?

Reply:
done